lib/test_queue/runner.rb

```diff
@@ -316,6 +311,8 @@ def distribute_queue
       remote_workers = 0
 
       until @queue.empty? && remote_workers == 0
+        queue_status(@start_time, @queue.size, @workers.size, remote_workers)
```

Comment:
Should we catch exceptions here and explicitly clean up if this raises? Is it possible to leave orphaned workers lying around?